tests/effectiveness/barrier_15_delay_100ms.self

```diff
@@ -1,4 +1,4 @@
-MIMPI_WRITE_DELAY=100 timeout 2s ./mimpirun 15 examples_build/bare_barrier
+DELAY=100 timeout 2s ./mimpirun 15 examples_build/bare_barrier
```

Comment:
Why not set this as `CHANNELS_WRITE_DELAY` and avoid subsequent fiddling with `setenv` in examples?

Reply:
The way it is done now, DELAY is set only for duration of function which execution time we want to measure. Therefore one can send some helper messages in MIMPI_Init or MIMPI_Finalise if one wants.